sdk/storage/Azure.Storage.Files/src/Models/FileHttpHeaders.cs

```diff
@@ -13,7 +13,6 @@ namespace Azure.Storage.Files.Models
     /// These properties are represented as standard HTTP headers use standard
     /// names, as specified in the Header Field Definitions section 14 of the
     /// HTTP/1.1 protocol specification.
-    /// 
```

Comment:
As we chatted about - I like these lines because they show up in intellisense and then it's clearer that folks can disregard after they've read the little top blurb.